src/shared/config/apiUrls.ts

```diff
@@ -76,6 +76,11 @@ const apiUrls = {
       '/publications'
     ),
   organismSuggester: '/uniprot/api/suggester?dict=organism&query=?',
+
+  // TODO: move that to UniRef-specific file?
```

Comment:
Initially thought `apiUrls` would contain a json object of namespaces (uniprot, uniref etc...), each containing all the urls specific to that namespace. However over time it's turned into something else so maybe having one per namespace is not a bad idea.

Reply:
It's a bit of a monster right now 😅

Reply:
In my mind it was going to be this beautifully formatted simple thing! 😆